packages/salesforcedx-vscode-soql/.vscodeignore

```diff
@@ -10,7 +10,6 @@ src/**
 **/*.map
 **/*.ts
 **/tsconfig.json
-**/*[!README].md
```

Comment:
This one line caused a huge amount of pain.  In core some of the message files are .md files so they can't be removed.